app/src/main/res/values/strings.xml

```diff
@@ -678,4 +678,6 @@ Upload your first media by tapping on the add button.</string>
   <string name="advanced_query_info_text">You can customize the Nearby query. If you get errors, reset and apply.</string>
   <string name="apply">Apply</string>
   <string name="reset">Reset</string>
+  <string name="turn_on_camera_location_title">Turn on location in your camera app\'s settings</string>
```

Comment:
Since it is just a title, could you please change this to just `Camera location`? Thanks!

Reply:
Or if it is possible to not put any title, that would be even better. The message below is already enough, I feel.